deepchem/feat/graph_data.py

```diff
@@ -2,73 +2,69 @@
 import numpy as np
 
 
-class MoleculeGraphData:
-  """MoleculeGraphData class
+class GraphData:
+  """GraphData class
 
   This data class is almost same as `torch_geometric.data.Data 
   <https://pytorch-geometric.readthedocs.io/en/latest/modules/data.html#torch_geometric.data.Data>`_.
 
```

Comment:
Could you add an example showing how to instantiate a `GraphData` class and extract its attributes?